less/forum/NotificationList.less

```diff
@@ -98,6 +102,25 @@
     font-weight: bold;
     text-transform: uppercase;
   }
+
+  .Notification-action {
+    float: right;
+    display: none;
+    margin-top: -11px;
+    margin-right: -11px;
+
+    & when (@config-colored-header = true) {
+      .Button--color(@control-color, @control-bg);
+
+      &:hover {
+        color: @link-color;
+      }
+    }
+
+    i.icon {
```

Comment:
I guess you dont' need the element selector here, just the class is enough and leaves our selector specificity more consistent.